client/src/pages/Info/index.tsx

```diff
@@ -45,16 +37,7 @@ const Info: FC = () => {
         {linkIcons.map(({ title, icon, name }) => {
           return (
             <div className={styles.iconBox}>
-              <a
-                key={name}
-                href={`#${name}`}
-                onClick={event => {
-                  return handleClick(event, name);
-                }}
-                onKeyDown={event => {
-                  return handleClick(event, name);
-                }}
-              >
+              <a key={name} href={`about/#${name}`}>
```

Comment:
`href={`#${name}`}` was correct

Reply:
No, this way I always go to the main page 
![Screenshot 2022-10-14 154415](https://user-images.githubusercontent.com/55662628/195850064-9b1ba12c-35e5-4f34-b709-d17c71d8af10.png)

Reply:
1. url `<path>/#<anchor>` is anyway incorrect, anchors have url `<path>#<anchor>`
2. for future: in most cases it should work as defined first time, but exactly in your case it won't work - https://stackoverflow.com/a/46674310

Reply:
It is working like this `href={`about#${name}`}`  :)  But somehow I can't get  smooth scroll behavior...

Reply:
I have tried to wrap all page, but scroll is the same both ways.

Reply:
Please read conversation in Vilius PR one more time, take a look at his code, attentively check and define all required additional css properties.

Reply:
I want to ask about api get.ts. Is it worth reducing the code further? Perhaps it would be possible to reuse the try catch block, but I don't think that it would significantly affect the number of lines of code. What do you think?

Reply:
get.ts file looks good for me, the only thing to refactor is types. i see in your function these return types and don't understand, why in one case you add ['data'] but in second one not. please provide reason or fix it
getHourlyWeather function: `Promise<AxiosResponse<HourlyWeather[]>['data'] | undefined>`
getDailyWeather function: `Promise<AxiosResponse<DailyWeather[]> | undefined>`

Reply:
I understand I have to use one method for consistency. I did it in different ways because I wanted you to tell me which way is better to use. Does it really matter?

Reply:
i was not talking about methods, i was talking about types

Reply:
I did not express myself correctly. Yes I was talking about types. The different ways I type data/ forecast/current
 1 )` Promise< AxiosResponse<AuthenticationResponse>['data'] | undefined`; and then `const result: AxiosResponse<AuthenticationResponse> = `
 2) another way I create export type response types in types.ts - `
LocationDataResponse = Response & {
  data: LocationData; 
` and then ` const result: AxiosResponse<LocationDataResponse> =` 
}; 
 3) third way I get it like this : Promise<AxiosResponse<HourlyWeather[]>['data']  and then  ' const result = await forecaClient.get<{ forecast: HourlyWeather[] }>'